packages/terra-application-docs/src/terra-dev-site/test/terra-application/primary-navigation-layout/PrimaryNavigationLayout5.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+import ApplicationBase from '@cerner/terra-application/lib/application-base';
+import ApplicationContainer from '@cerner/terra-application/lib/application-container';
+import PrimaryNavigationLayout, { NavigationItem } from '@cerner/terra-application/lib/primary-navigation-layout';
+
+import PrimaryNavigationWorkspace from '@cerner/terra-application/lib/primary-navigation-layout/workspace-layout/PrimaryNavigationWorkspace';
+import PrimaryNavigationWorkspaceItem from '@cerner/terra-application/lib/primary-navigation-layout/workspace-layout/PrimaryNavigationWorkspaceItem';
+import { WorkspaceContent } from '@cerner/terra-application/lib//workspace';
+
+import TestPage from '../shared/TestPage';
+
+const Tab1 = () => (
+  <WorkspaceContent>
+    <p>Wombat 1</p>
+  </WorkspaceContent>
+);
+
+const Tab2 = () => (
+  <WorkspaceContent>
+    <p>Wombat 2</p>
+  </WorkspaceContent>
+);
+
+const PrimaryNavigationLayout5 = () => {
+  const [
+    activeNavigationItemKey,
+    setActiveNavigationItemKey,
+  ] = React.useState('1');
+
+  const wrapper = (
```

Comment:
nit picky but maybe a touch more clear?
```suggestion
  const workspace = (
```

Reply:
updated.